src/components/types.ts

```diff
@@ -1,3 +1,7 @@
 export interface BaseComponentProps {
   className?: string;
 }
+
+export interface TestId {
```

Comment:
Hey @frano-m! Checking in on the naming convention here: do you normally add a `Props` suffix for props interfaces?

Reply:
Sorry @MillenniumFalconMechanic I was using the ticket verbatim. I'll add Props to the end.